### ISSUE TITLE
Write a guide with zero assumptions

### ISSUE THREAD

**Issue by fire** [HashEdits/BFI_VRCFT_Module#1]
Start from someone getting the required equipment and end with vrchat chatting with brain emotes.

**Comment by fire** [comment]
Required equipment:

- muse s (or any other supported brain sensing equipment supported by BrainFlows) https://choosemuse.com/products/muse-s-gen-2
- computer with a gpu / accelerator that can train and play vrchat at acceptable frame rates
- vr hmd (optional)

Required software:
- https://github.com/ChilloutCharles/BrainFlowsIntoVRChat
- https://store.steampowered.com/app/438100/VRChat/
- https://github.com/HashEdits/BFI_VRCFT_Module
- https://github.com/benaclejames/VRCFaceTracking

Edit me @HashEdits

**Comment by fire** [comment]
I'll migrate this to wiki if I Can figure out how to do it.

**Comment by fire** [comment]
Emote table to train:

| Expression  | Reference   | Description                                  |
|-------------|-------------|----------------------------------------------|
| Nothing     |             | Nothing done at all                          | 
| Eyeclosed   | eyeclosed   | Eyelids closing fully                        |
| Smile       | smile       | Smile with mouth opened                      |
| Frown       | frown       | N-shaped frown                               |
| Anger       | anger       | Brow going down                              |
| Cringe      | cringe      | Mouth stretches and reveals the bottom teeth |
| Cheekpuff   | cheekpuff   | Cheeks puffing out                           |
| Apeshape    | apeshape    | Lower jaw juts forward while keeping mouth closed |

**Comment by HashEdits** [comment]
> Required equipment:
> 
> * muse s (or any other supported brain sensing equipment supported by BrainFlows) https://choosemuse.com/products/muse-s-gen-2
> * computer with a gpu / accelerator that can train and play vrchat at acceptable frame rates
> * vr hmd (optional)
> 
> Required software:
> 
> * https://github.com/ChilloutCharles/BrainFlowsIntoVRChat
> * https://store.steampowered.com/app/438100/VRChat/
> * https://github.com/HashEdits/BFI_VRCFT_Module
> * https://github.com/benaclejames/VRCFaceTracking
> 
> Edit me @HashEdits

Good idea! will add something that links to brainflows's doccumentation on compatible headbands and something that says that twidge's already confirmed that it works great with the MuseS&2

**Comment by HashEdits** [comment]
Added a tested hardware section!

**Comment by fire** [comment]
@HashEdits mentioned that having a specific guide for one of the easier gestures would be helpful.

Like Eyeclosed.
